packages/browser/jest.env.js

```diff
@@ -0,0 +1,15 @@
+const Environment = require('jest-environment-jsdom');
+
+// Looks like jsdom does not support global TextEncoder/TextDecoder
+// https://github.com/jsdom/jsdom/issues/2524
+
+module.exports = class CustomTestEnvironment extends Environment {
+  async setup() {
+    await super.setup();
+    if (typeof this.global.TextEncoder === 'undefined') {
+      const { TextEncoder, TextDecoder } = require('util');
+      this.global.TextEncoder = TextEncoder;
+      this.global.TextDecoder = TextDecoder;
```

Comment:
Do think it'll simplify things if we just define these for the tests that need them instead of globally?

Reply:
Yep, will see if I can revert this!

Reply:
Looks like the alternative is to pass `transportOptions.textEncoder: new TextEncoder()` to every single test in:
`packages/browser/test/unit/index.test.ts`

These tests seem to be serialising an envelope to send a session:
```
   ReferenceError: TextEncoder is not defined
    46 |  */
    47 | export function serializeEnvelope(envelope: Envelope, textEncoder?: TextEncoderInternal): string | Uint8Array {
  > 48 |   const utf8 = textEncoder || new TextEncoder();
    49 |
    50 |   const [envHeaders, items] = envelope;
    51 |
  at Object.utils_1 [as serializeEnvelope] (../utils/src/envelope.ts:48:35)
  at taskProducer (../core/src/transports/base.ts:72:27)
  at Object.add (../utils/src/promisebuffer.ts:49:18)
  at Object.send (../core/src/transports/base.ts:84:19)
  at BrowserClient._sendEnvelope (../core/src/baseclient.ts:675:23)
  at BrowserClient.sendSession (../core/src/baseclient.ts:306:12)
  at BrowserClient.captureSession (../core/src/baseclient.ts:203:12)
  at Hub._sendSessionUpdate (../hub/src/hub.ts:452:16)
  at Hub.endSession (../hub/src/hub.ts:400:10)
  at Hub.startSession (../hub/src/hub.ts:433:12)
  at startSessionOnHub (src/sdk.ts:245:7)
  at startSessionTracking (src/sdk.ts:277:3)
  at Object.init (src/sdk.ts:121:5)
  at Object.<anonymous> (test/unit/index.test.ts:257:7)
```
Looks like the previous test starts the session and each subsequent test records a session end and then start. These should probably have `autoSessionTracking: false`.

Reply:
Yeah I think we can leave it then.